tests/volume/gluster-volume-set.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- hosts: servers
+  tasks:
+    - name: Toggle volume options
+      volume: action=set
```

Comment:
This is not needed in this patch set I think. Any reason why you added this in this patch?